draft-packages/stories/RatingScaleLegacy.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react"
+
+import {
+  RatingScaleLegacy,
+  Scale,
+  ScaleItem,
+} from "@kaizen/draft-rating-scale-legacy"
+
+export default {
+  title: "RatingScaleLegacy (React)",
+  component: RatingScaleLegacy,
+  parameters: {
+    info: {
+      text: `
+        import { RatingScale } from "@kaizen/draft-rating-scale-legacy";
+      `,
+    },
+  },
+}
+
+const scale: Scale = [
+  {
+    value: 1,
+    label: "Very poor",
+  },
+  {
+    value: 2,
+    label: "Poor",
+  },
+  {
+    value: 3,
+    label: "Fair",
+  },
+  {
+    value: 4,
+    label: "Good",
+  },
+  {
+    value: 5,
+    label: "Very good",
+  },
+]
+
+export const DefaultStory = () => {
+  const [selectedItem, setSelectedItem] = useState<ScaleItem | null>(null)
+
+  return (
+    <div style={{ margin: "48px 24px" }}>
+      <RatingScaleLegacy
+        scale={scale}
+        questionId="123"
+        labelId="456"
```

Comment:
I know it's a pain to style, but it would be good to have an actual label in these examples which is linked here, to document how this prop is meant to be used.